anyway/flask_app.py

```diff
@@ -1378,19 +1378,13 @@ def get_dict_file(directory):
         yield name, df
 
 
-@app.route("/markers/polygon/", methods=["GET"])
-def acc_in_area_query():
+@app.route("/api/markers/polygon/<string:pol_str>", methods=["GET"])
```

Comment:
looks great!
However, I don't want it to be an API yet, since such query can cause a big load on our db.
Can you remove the API and keep the `get_acc_in_area_query` function for now? We will soon use it internally